cdap-unit-test/src/main/java/co/cask/cdap/test/TestBase.java

```diff
@@ -298,6 +300,8 @@ protected void configure() {
     dsOpService.startAndWait();
     datasetService = injector.getInstance(DatasetService.class);
     datasetService.startAndWait();
+    namespaceService = injector.getInstance(NamespaceService.class);
```

Comment:
``DefaultStoreTest`` was failing because it clears the MDS before every run, so the *default* namespace gets deleted, hence deploy returns ``null``. Fixed that temporarily with a TODO.